src/Netflix/Netflix.ts

```diff
@@ -0,0 +1,760 @@
+// lipsurf-plugins/src/Netflix/Netflix.ts
+/// <reference types="lipsurf-types/extension"/>
+declare const PluginBase: IPluginBase;
+
+// Messaging
+
+const LIPSURF_BOOT_SCRIPT_ID = "lipsurf-netflix-script";
+const TO_PAGE_PROOF_KEY = String(Math.floor(Math.random() * 1000));
+const FROM_PAGE_PROOF_KEY = String(Math.floor(Math.random() * 1000));
+
+type Message<T> = { proofKey: string; payload: T };
+
+const sendMessage = (payload: Command) =>
+  (window as any).postMessage(
+    JSON.stringify({ proofKey: TO_PAGE_PROOF_KEY, payload } as Message<Command>)
+  );
+
+/**
+ * Receive Message
+ */
+(() => {
+  window.addEventListener("message", ev => receiveMessage(ev.data));
+  const receiveMessage = (messageStr: string) => {
+    try {
+      const message = JSON.parse(messageStr) as Message<Command>;
+      if (message.proofKey === FROM_PAGE_PROOF_KEY) {
+        const command = message.payload;
+        switch (command.key) {
+          case "changeText":
+            return handleChangeTextAnswer(command);
+          case "changeAudio":
+            return handleChangeAudioAnswer(command);
+        }
+      }
+    } catch {
+      // Silent catch
+      // This try catch is used only to catch JSON.parse failure
+      // Which is impossible to happen within Netflix Plugin scope
+    }
+  };
+})();
+
+// Utilities
+
+const stripNonAlphaNumericAndTrim = (someString: string) =>
+  someString
+    .split(" ")
+    .map(someString => someString.replace(/[^\w\s]/gi, "").trim())
+    .join(" ");
+
+const navigateToSearch = (title: string) => {
+  window.location.href = `https://www.netflix.com/search?q=${encodeURIComponent(
+    title
+  )}`;
+};
+
+// Context
+
+/**
+ * Context is currently disabled becacuse some weird behavior with non-normal
+ * context
+ */
+
+enum NetflixPluginContextEnum {
+  watch = "Netflix Video Player Controls",
+  browse = "Browse Netflix"
+}
+type NetflixPluginContext = NetflixPluginContextEnum | null;
+
+const contextManager = (() => {
+  let enabled = true;
+
+  type ContextPatcher = {
+    add: (context: string) => void;
+    remove: (context: string) => void;
+  };
+
+  /**
+   * Patch context to avoid context duplication
```

Comment:
Ah, okay. I forgot to tell you this. Removing this.

Reply:
On second, thought. I will keep this until the fix is done, primarily because I think double-fixes are better than no-fixes, even temporarily, secondly I'm still working on it so I need it working. I'll remove it when the fix is there.